join-chat-to-teams-meeting/README.md

```diff
@@ -0,0 +1,65 @@
+---
+page_type: sample
+languages:
+- java
+products:
+- azure
+- azure-communication-services
+---
+
+# Quickstart: Join your chat app to a Teams meeting
+
+For full instructions on how to build this code sample from scratch, look at [Quickstart: Join your chat app to a Teams meeting](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/chat/meeting-interop?pivots=platform-android)
+
+## Prerequisites
+
+To complete this tutorial, you’ll need the following prerequisites:
+
+- An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
+- [Android Studio](https://developer.android.com/studio), for running your Android application.
+- A deployed Communication Services resource. [Create a Communication Services resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource).
+- A [User Access Token](https://docs.microsoft.com/azure/communication-services/quickstarts/access-tokens) for your Azure Communication Service.
```

Comment:
A Teams meeting link and Meeting/chat Thread ID?

Reply:
The thread ID is generated as part of the Teams meeting. This is wrapped up in the Teams deployment below.